postman/package.json

```diff
@@ -10,6 +10,7 @@
   "devDependencies": {},
   "scripts": {
     "build_202108-slice-demo": "node ./202108-slice-demo/index.js",
+    "build_202110-milmob-4": "node ./202110-milmob-4/index.js",
```

Comment:
Looks like I missed adding this for the Sept slice demo. Thanks for adding this one.